script.js

```diff
@@ -0,0 +1,257 @@
+
+////////////1///////////////
+
+let result = prompt ('Ваше имя?', '');
+alert (`Ваше имя ${result}`);
+
+const sayHello = function(name) {
+    if (name === "Mark") {
```

Comment:
what if I call your function with 'mark' (lower cased first letter)? 
if you want to compare strings the best way is to make values inside comparison in one register lower or upper and only after that compare them